core/trino-main/src/main/java/io/trino/sql/planner/RelationPlanner.java

```diff
@@ -251,26 +251,30 @@ protected RelationPlan visitTable(Table node, Void context)
             }
         }
 
-        plan = addRowFilters(node, plan);
+        plan = addRowConstraints(node, plan);
```

Comment:
As I mentioned above, row filters and check constraints are two difference concepts. Let's not mix them up.